resolvers/webpack/.babelrc.js

```diff
@@ -0,0 +1 @@
+../../.babelrc.js
```

Comment:
I prefer non-js configs only if possible.

Reply:
@ljharb I removed this file and the `mocharc` file, so I resolve this conversation, so I allow myself to resolve here.